CHANGELOG_PENDING.md

```diff
@@ -19,6 +19,7 @@
 #### Broadcaster
 
 - \#1782 Fix SegsInFlight data-loss on refreshing O sessions (@darkdragon)
+- \#1814 Add price checks when caching orchestrator responses during discovery
```

Comment:
Think it is missing an author here , looks good otherwise !

Reply:
Heh realized right after I reviewed your PR 😆 

Fixed in c4cefe3